README.rst

```diff
@@ -5,6 +5,10 @@ BrotliCFFI
     :alt: Version
     :target: https://pypi.org/project/brotlicffi
 
+.. image:: https://anaconda.org/conda-forge/brotlicffi/badges/version.svg
+    :alt: Version-Conda
+    :target: https://anaconda.org/conda-forge/brotlicffi
+
```

Comment:
Can we also add instructions on how to install from Conda Forge like so:

`$ conda install -c conda-forge brotlicffi`